astro/src/content/docs/lifecycle/migrate-users/bulk/stytch.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: Migration From Stytch
+description: How to migrate your users from Stytch to FusionAuth.
+section: lifecycle
+subcategory: migrate users
+tertcategory: bulk
+prerequisites: Docker, Node
+importCodeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-import-scripts/master
+javaCodeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-contrib/master
+---
+
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import InlineField from 'src/components/InlineField.astro';
+import Aside from '/src/components/Aside.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import AdditionalSupport from 'src/content/docs/lifecycle/migrate-users/bulk/_additional-support.mdx';
+
+## Overview
+
+This article explains how to import users from Stytch into FusionAuth. It is a low-level, technical tutorial focusing on transferring password hashes, calling APIs, and preparing data. The article does not explain the high-level process of a migration strategy. This is explained in the [migration overview article](/docs/lifecycle/migrate-users/bulk/general). Before proceeding, please ensure you understand concepts like dual write, backfill, final synchronization, rolling migration, and cutover. The [Stytch documentation on migration strategies](https://stytch.com/docs/guides/migrations/migrating-user-data-statically) is also useful reading. Be aware of all laws regarding the protection and transfer of personal information in your country.
+
+Finally, if you want to import user passwords in addition to user personal details, you need a basic understanding of how password hashing and salts work. FusionAuth has a [hashing article](/articles/security/math-of-password-hashing-algorithms-entropy) that is a good starting point.
+
+### Limitations
+
+- This article explains how to migrate Stytch users from a Consumer Authentication project. It has not been tested with their B2B Saas Authentication project type.
+- This article explains how to import passwords into FusionAuth, but does not deal with other Stytch authentication types: magic links, passkeys, passcodes, mobile biometrics, two-factor authentication, and social logins such as Google OAuth.
```

Comment:
We do have a component that talks about social logins
import SocialLoginNote from 'src/content/docs/lifecycle/migrate-users/bulk/_social-login-note.mdx';

There are a few other components you may want to include. Please check out the Cognito migration guide and take a look at some of the following:
MappingUserAttributes
OtherEntitiesIntro
Identifiers
and others.